src/overview/components/Overview.jsx

```diff
@@ -4,10 +4,23 @@ import PropTypes from 'prop-types'
 import Header from './Header'
 import DeleteConfirmation from './DeleteConfirmation'
 import styles from './Overview.css'
+import Filters from './Filters'
 
 const Overview = props => [
     <Header key="head" {...props} />,
-    <div key="body" className={styles.main}>
+    <div className={styles.filtersContainer} key="filters">
+        {props.showFilter && (
```

Comment:
Maybe better to always render `<Filters>` and just apply a special class to it if `props.showFilter` is set. So handling it all in CSS. Then we could put a nice transition on it or something later. Not 100% important, but something to think about :)